internal/util/pidlimit.go

```diff
@@ -41,7 +41,7 @@ func getCgroupPidsFile() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	defer cgroup.Close()
+	defer cgroup.Close() // #nosec: error on close is not critical here
```

Comment:
how about handle log error in defer function here and in other places  as we dont need to handle nosec

Reply:
I did not do so, because we do not care about the error here. The function worked with closing the file returning an error.